invenio_rdm_records/config.py

```diff
@@ -261,6 +269,18 @@ def always_valid(identifier):
         "mostviewed",
         "mostdownloaded",
     ],
+    "query_parser_cls": QueryParser.factory(
+        mapping={
+            "internal_notes.note": RestrictedTerm(administration_permission),
+            "internal_notes.id": RestrictedTerm(administration_permission),
+            "internal_notes.added_by": RestrictedTerm(administration_permission),
+            "internal_notes.timestamp": RestrictedTerm(administration_permission),
+            "_exists_": RestrictedTerm(
```

Comment:
this requires some explanations :)

Reply:
I am not sure if you added the comment before we talked or after :) we can rediscuss if it was after!